services/has-many-getter.js

```diff
@@ -117,7 +113,17 @@ function HasManyGetter(model, association, opts, params) {
           fieldsSearched = searchBuilder.getFieldsSearched();
         }
 
-        return [records, count, fieldsSearched];
+        records = _.slice(records, getSkip(), getSkip() + getLimit());
+
+        return [records, fieldsSearched];
+      });
+  };
+
+  this.count = function () {
+    return getRecords()
+      .then(function (records) {
+        count = records.length;
+        return count;
```

Comment:
`return records.length;` is lighter.

Can you improve this code to prevent the whole records count in Javascript?

Reply:
As discussed  regarding the improvement of the `records.length`, not as easy as I thought, can we do it in another iteration ?